src/StudyUtils.in.jsm

```diff
@@ -22,6 +44,10 @@ const CID = Cu.import("resource://gre/modules/ClientID.jsm", null);
 const { TelemetryController } = Cu.import("resource://gre/modules/TelemetryController.jsm", null);
 const { TelemetryEnvironment } = Cu.import("resource://gre/modules/TelemetryEnvironment.jsm", null);
 
+/**
+* Gets the telemetry client ID for the user (AKA their Firefox user ID).
```

Comment:
Don't call it a Firefox User ID.  WE do not track users, and they do not have IDs.  It is a Telemetry Client ID

Reply:
Besides the optics issue, there are other uncorrelated IDs that clients have (like for Shield and Pioneer)

Reply:
@mythmon , what do you mean by "optics"? I removed the aside. For my own understanding, what is the telemetry client ID based on? Is it unique to the user's browser?